Source/Scene/Cesium3DTileset.js

```diff
@@ -572,6 +572,31 @@ define([
         },
 
         /**
+         * The tileset's bounding volume. As previously accessed by tileset._root.contentBoundingVolume.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {TileBoundingVolume}
+         * @readonly
+         *
+         * @exception {DeveloperError} The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.
+         *
+         * @example
+         * console.log('Bounding volume of tileset: ' + tileset.boundingVolume);
+         */
+        boundingVolume : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
+                }
+                //>>includeEnd('debug');
+                
+                return this._root.contentBoundingVolume;
```

Comment:
I'm changing my mind here, but `return this._root._boundingVolume` is probably a better choice.